gulpfile.js

```diff
@@ -18,33 +18,36 @@ combine_js = function (target) {
   console.log("compile "+target+" coffee files")
   return gulp.src(['./src/'+target+'/*.coffee','./src/*.coffee'])
     .pipe(coffee({bare: true}).on('error', gutil.log))
+
 };
 
-static_stream = function(){
-  return gulp.src (['css/*','js/*.js','img/*.png'],{ base: "./static", cwd : "./static"})
+// ninja foo funtion to return function based on destination directory
+static_stream = function(dest) {
+  this.dest = dest
+  return (
+    function(_this) {
+      return function() {
+        return gulp.src(['css/*', 'js/*.js', 'img/*.png'], {
+          base: "./static",
+          cwd: "./static"
+        }).pipe(gulp.dest('./' + _this.dest ))
+      }
+    }
+  )(this)
```

Comment:
Guilty of ninja foo as charged but gulp needs a function (1level) and I memoized the destination directory (2 levels).